lms/templates/base.html

```diff
@@ -9,6 +9,12 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='my.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='prism.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='hawcons.css') }}">
+    <link rel="manifest" href="{{ url_for('static', filename='manifest.json') }}">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="transparent">
+    <link rel="icon" sizes="16x16 32x32 48x48"  href="{{ url_for('static', filename = 'icons/icons.ico')}}">
```

Comment:
In Python we don't put spaces before and after the equal sign. Please trim them :)